mage_ai/settings/__init__.py

```diff
@@ -19,10 +19,18 @@
     DISABLE_NOTEBOOK_EDIT_ACCESS = 1 if os.getenv('DISABLE_NOTEBOOK_EDIT_ACCESS') else 0
 
 
+
+
 def is_disable_pipeline_edit_access():
     return DISABLE_NOTEBOOK_EDIT_ACCESS >= 1
 
 
+#------------------------- DISABLE TERMINAL ----------------------#'
+
+DISABLE_TERMINAL = 1 if os.environ.get("DISABLE_TERMINAL") == "1" else 0
```

Comment:
you can use `os.getenv('DISABLE_TERMINAL')`

Reply:
You also don't need the `is_terminal_access_disable()` method for this variable. You can just fetch the value of the variable.

Reply:
I change os.environt to getenv(). 
Remove the function, but in the condition still ask if the variable is 1 so it only applies with that value and ignore in other cases.

Reply:
I would change the definition to how it is for the `REQUIRE_USER_AUTHENTICATION`, so that you can just check if the variable is true or false.

Reply:
I made the suggested changes